basic.md

```diff
@@ -702,6 +702,17 @@ Currently not implemented, does nothing
 
 Currently not implemented, does nothing
 
+### `END`
+
+End execution of the program silently. Note that `SYSTEM` seems to exhibit the same behaviour.
+
+```basic
```

Comment:
As you pointed out, there is no TeleBASIC language extension on GitHub, so this tag is extraneous.  No other code block in this document has a language extension tag, so I would remove this (at least until someone makes a language extension!)

Reply:
... at which point it would be good to reformat all the example code blocks and split them up into "program" and "result" blocks.

https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#adding-an-extension-to-a-language

Reply:
The trouble is that TeleBASIC is a fairly obscure flavour of BASIC and none of our strange additions are highlighted, such as the `TH_` functions/keywords.  These blocks would be highlighted but none of the others would, and so to stay consistent I would say that the tag should be removed.

Reply:
Ok, fair enough.